src/defaultOptions/ScreenSize.ts

```diff
@@ -7,7 +7,7 @@ export enum ScreenSize {
   tablet = 'tablet',
```

Comment:
Random thought. Does `mini`, `x`, and `max` make sense on android? Maybe this would be another good case for the utility classes. Or being able to specify size -> name aliases

Reply:
Agreed. Probably not the right _default_ options. Maybe something like `smPhone`, `phone`, `lgPhone`, `tablet`, `smDesktop`. Definitely could be re-thought a bit. Maybe there is a generic way to achieve this. The difference between these and custom utilities is that these are always valid, but are "switched off" on other platforms.